src/main/cpp/t07_ip.cpp

```diff
@@ -29,10 +29,52 @@
 
 #include "t07_ip.h"
 #include <iostream>
+#include <string>
 
 
 using namespace std;
 
 int t07_ip() {
+	string s, s2;
+	getline(cin, s);
+	int size = s.size();
+	int a = -1;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'a' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4721051)